db/migrate/20230222155433_create_posts.rb

```diff
@@ -0,0 +1,15 @@
+class CreatePosts < ActiveRecord::Migration[7.0]
+  def change
+    create_table :posts do |t|
+      t.integer :authorId
```

Comment:
- [ ] Please 🙏🏽 correct the column type you should use references 👉🏽  `t.references : author, foreign_key: { to_table: 'users' `}` this will create author_id and a corresponding index.